compiler/optimizer/Inliner.cpp

```diff
@@ -6291,8 +6330,8 @@ void TR_InlinerTracer::dumpPrexArgInfo(TR_PrexArgInfo* argInfo)
       if (arg && arg->getClass())
          {
          char* className = TR::Compiler->cls.classSignature(comp(), arg->getClass(), trMemory());
-         traceMsg( comp(),  "<Argument no=%d address=%p classIsFixed=%d classIsPreexistent=%d class=%p className= %s/>\n",
-         i, arg, arg->classIsFixed(), arg->classIsPreexistent(), arg->getClass(), className);
+         traceMsg( comp(),  "<Argument no=%d address=%p classIsFixed=%d classIsPreexistent=%d argIsKnownObject=%d class=%p className= %s/>\n",
```

Comment:
if this is printing the known object index should the trace reflect that?

Reply:
Added code to print the known object index.